### PR TITLE
feat!: v8

### PR DESCRIPTION
**Pull request by bernardcooke53**
With much love to the whole community :heart: 

Open for comments/review, otherwise I think this is ready to merge. Addresses #487

### CODE REVIEW

**Comment by MicaelJarniac** [comment]
Is there a way to deploy the docs of this upcoming version before it's released?

**Comment by bernardcooke53** [comment]
Not that I'm aware of - might need to ask @relekang (I don't have access to the pypi or readthedocs stuff)

**Comment by MicaelJarniac** [comment]
Don't worry then. I asked because it can sometimes be useful to preview the docs, so it's easier to look for errors or to learn how the new version works.

But it's still possible to view the Markdown files directly here on the repo, or to render the docs locally, so this isn't a big deal.

File: docs/changelog_templates.rst [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+  (See :ref:`changelog-templates-template-rendering-template-context-release-history`)
+* ``filters: Tuple[Callable[..., Any], ...]``: a tuple of filters for the template environment.
+  These are added to the environment's ``filters``, and therefore there should be no need to
+  access these from the ``context`` object inside the template.
+
+Currently, two filters are defined:
+
+* ``pull_request_url: Callable[[str], str]``: given a pull request number, return a URL
+  to the pull request in the remote.
+* ``pull_request_url: Callable[[str], str]``: given a commit hash, return a URL to the
```
Comment (afuetterer) [inline docs/changelog_templates.rst:169]: Should this be `commit_hash_url` instead?

Comment (MicaelJarniac) [reply docs/changelog_templates.rst:169]:
```suggestion
* ``commit_hash_url: Callable[[str], str]``: given a commit hash, return a URL to the
```

**Pull request merged**